tests/Tests/400_ods_dashboard/400_ods_dashboard.robot

```diff
@@ -33,6 +33,24 @@ Verify Content In RHODS Explore Section
     Check Number Of Displayed Cards Is Correct  expected_data=${EXP_DATA_DICT}
     Check Cards Details Are Correct   expected_data=${EXP_DATA_DICT}
 
+Verify CSS Style Of Getting Started Descriptions
+    [Documentation]    Verifies the CSS style is not changed. It uses JupyterHub card as sample
```

Comment:
Is it enough to check only the JH card? Should it check all cards?

Reply:
I think it is enough, because the same CSS style is applied to all the cards and it is stored in a external sheet. So I've identified a couple of elements to check (code snippet, <h1> title, paragraph) and JH has all of these. So if some CSS rules about one of these elements got corrupted, we catch it and it is reasonable all the cards have the same problems.